Lib/test/test_list.py

```diff
@@ -163,6 +163,31 @@ class L(list): pass
         with self.assertRaises(TypeError):
             (3,) + L([1,2])
 
+    def test_equal_operator_modifying_operand(self):
+        # test fix for seg fault reported in issue 38588 part 2.
```

Comment:
Please, use `bpo-38588` istead of `issue 38588`. This is to prevent confusion when
we move issue tracker.